.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Should we do this on every commit or tags only?

Reply:
I would say Tags, as it keeps releases intentional. Triggering on every commit offers faster updates but risks noise and less meaningful versions.  I am open to other opinions as well!

Reply:
While I generally favor solutions that are fully automated, I think it might be a good idea to use tags for releases. Otherwise, we must somehow ensure that each commit also includes a version bump. And if we handle it well, I think it would be easier for people to find a specific version of the code matching the chart version they use.

Reply:
I agree. I only put it to master because that's what they have in the  docs ☺️